### PR TITLE
Bereket.mobilefoldersetup

### PR DESCRIPTION
**Pull request by BekyH**

### CODE REVIEW

File: Mobile/dark_knights/lib/features/example/domain/repositories/exaample_repo..dart
Comment (Redi22) [inline Mobile/dark_knights/lib/features/example/domain/repositories/exaample_repo..dart:1]: Can we rename this to example_repository?

File: Mobile/dartsmiths/lib/features/example/domain/repositories/exaample_repo..dart
Comment (Redi22) [inline Mobile/dartsmiths/lib/features/example/domain/repositories/exaample_repo..dart:1]: rename to example_repository

File: Mobile/matador/lib/features/example/domain/repositories/exaample_repo..dart
Comment (Redi22) [inline Mobile/matador/lib/features/example/domain/repositories/exaample_repo..dart:1]: rename to example_repository

File: Mobile/matador/lib/features/example/data/repositories/example_repo_impl.dart
Comment (Redi22) [inline Mobile/matador/lib/features/example/data/repositories/example_repo_impl.dart:1]: rename to example_repository_impl. Same applies for the other 2 similar files in the other projects

**Review by Redi22 — APPROVED** [review]

**Review by YiseBoge — CHANGES_REQUESTED** [review]
I would approve this but for the sake of consistency i would aks the project names be renamed in the format of "starter_project_[team name]"

**Review by YiseBoge — DISMISSED** [review]

**Review by lealemgetahun — DISMISSED** [review]

**Review by lealemgetahun — APPROVED** [review]

**Review by YiseBoge — APPROVED** [review]

**Pull request merged**